geoapps/inversion/potential_fields/application.py

```diff
@@ -875,7 +875,13 @@ def inversion_type_observer(self, _):
             self._param_class = GravityParams
 
         self.params = self._param_class(validate=False, verbose=False)
-        self.ga_group_name.value = self.params.out_group
+
+        if getattr(self.params, "_out_group", None) is not None:
+            self.ga_group_name.value = self.params.out_group.name
+        else:
+            self.ga_group_name.value = (
```

Comment:
Don't think this is needed since you put it in the base params property.  Here and in the other applications.

Reply:
I just don't want to change the rest of the behaviour now. Otherwise I need to remove those widgets, and that's bigger.